WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -103,6 +103,10 @@ def index
       end
     end
 
+    if (1..4).cover? params[:days].to_i
```

Comment:
Sure, I have this in mind. But on the other hand I believe that we can pick the most suitable at this moment.